.travis.yml

```diff
@@ -1,6 +1,6 @@
 # Declare python as our language. This way we get our chosen Python version,
 # and pip is available. Gcc and clang are available anyway.
-distro: xenial
+dist: focal
 os: linux
 language: python
 python: 3.5
```

Comment:
Maybe move to a more modern python version (at least 3.8, or I suggest 3.10 which is the one in Ubuntu 22.04 jammy)? This could save us some trouble if the packages break with 3.5.